Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/d/Library/Python/3.11/lib/python/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
//cls 
qui {
	if 1 {
//1. data
global repo "https://github.com/jhustata/project/raw/main/"
global nhanes "https://wwwn.cdc.gov/Nchs/Nhanes/"

//2. code
do ${repo}followup.do
save ~/documents/github/statatwo/followup, replace 
import sasxport5 "${nhanes}1999-2000/DEMO.XPT", clear
merge 1:1 seqn using followup, nogen
save ~/documents/github/statatwo/survey_followup, replace 

//3. parameters
import sasxport5 "${nhanes}1999-2000/HUQ.XPT", clear
tab huq010 
merge 1:1 seqn using ~/documents/github/statatwo/survey_followup, nogen keep(matched)
rm ~/documents/github/statatwo/followup.dta
rm ~/documents/github/statatwo/survey_followup.dta
g years=permth_int/12
stset years, fail(mortstat)
replace huq010=. if huq010==9
label define huq 1 "Excellent" 2 "Very Good" 3 "Good" 4 "Fair" 5 "Poor"
label values huq010 huq 
levelsof huq010, local(numlevels)
local i=1
foreach l of numlist `numlevels' {
    local vallab: value label huq010 
	local catlab: lab `vallab' `l'
	global legend`i' = "`catlab'"
	local i= `i' + 1
}
save ~/documents/github/statatwo/week7, replace 
#delimit;
sts graph,  
    by(huq010)  
	fail  
	per(100) 
	ylab(0(20)80 ,  
	    format(%2.0f)  
	)  
	xlab(0(5)20)  
	tmax(20)  
	ti("Self-Reported Health and Mortality")  
	legend(  
	    order(5 4 3 2 1)  
		lab(1 "$legend1")  
		lab(2 "$legend2")  
		lab(3 "$legend3") 
		lab(4 "$legend4")  
		lab(5 "$legend5")  
		ring(0) pos(11) 
	) ;
#delimit cr
graph export ~/documents/github/statatwo/nonpara.png, replace 
save ~/documents/github/statatwo/wk8, replace 
    }
}
use ~/documents/github/statatwo/wk8, clear 
stcox i.huq010, basesurv(s0)
sort _t s0
list _t s0 in 1/20
keep _t s0
save ~/documents/github/statatwo/s0, replace 
------------------

----- stdout -----

file demo/NHANES_1999_2000_PUF.dta could not be opened
----- stderr -----
r(603);
----- stderr -----
r(603);
----- stdout -----

------------------



